### ISSUE TITLE
Add Chat API option allowing users to request human assistance

### ISSUE THREAD

**Issue by charludo** [digitalfabrik/integreat-cms#2826]
### Motivation

In the future, the chat functionality of the integreat app might involve functionality beyond conversation with region managers/counselors. In that case, users still might want to ensure they are talking to a human.

### Proposed Solution

Add a field `request_counselor` or similar to the POST body. When present, automatic responses (help messages, LLMs,...) should not be invoked, rather a ticket should be opened via the Zammad API.

### Alternatives

none

### User Story

As an app user, I want to be able to request human help if automated/LLM answers are indufficient.

### Additional Context

Blocked by 
- [x] #2619

**Comment by charludo** [comment]
@svenseeberg this can be closed, right?

**Comment by svenseeberg** [comment]
Yes, this status field already exists in the Zammad ticket object. For reference: https://github.com/digitalfabrik/integreat-chat/issues/25 and https://github.com/digitalfabrik/integreat-chat?tab=readme-ov-file#zammad-integration
